From 824156d669df4b8e96c3cd3b7db87293c0e6b879 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Thu, 10 May 2018 08:54:18 +0900
Subject: [PATCH 2/2] Add set_endian function

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
---
 build_config.rb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build_config.rb b/build_config.rb
index 7f52b5ee..927afd87 100644
--- a/build_config.rb
+++ b/build_config.rb
@@ -96,6 +96,7 @@ MRuby::Build.new('host-debug') do |conf|
   end
 
   enable_debug
+  set_endian
 
   # include the default GEMs
   conf.gembox 'default'
@@ -119,6 +120,7 @@ MRuby::Build.new('test') do |conf|
   end
 
   enable_debug
+  set_endian
   conf.enable_bintest
   conf.enable_test
 
-- 
2.17.1

