From 06dcd89d66f64cb963d9c811d82555caf9b2da85 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Sat, 5 Jan 2019 06:42:09 +0900
Subject: [PATCH 2/3] Add check function for endian

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 lib/mruby/build.rb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/lib/mruby/build.rb b/lib/mruby/build.rb
index 64a9269a..310127e8 100644
--- a/lib/mruby/build.rb
+++ b/lib/mruby/build.rb
@@ -119,6 +119,14 @@ module MRuby
       @enable_debug = true
     end
 
+    def set_endian
+      if "\x01\00".unpack('S')[0] == 1
+        little_endian
+      else
+        big_endian
+      end
+    end
+
     def big_endian
       if @endian
         puts "Endian has already specified as #{@endian}."
-- 
2.20.1

