diff --git a/lib/mruby/build.rb b/lib/mruby/build.rb
index 7d6aa49e..e2538204 100644
--- a/lib/mruby/build.rb
+++ b/lib/mruby/build.rb
@@ -104,7 +104,7 @@ module MRuby
       compilers.each do |c|
         c.defines += %w(MRB_DEBUG)
         if toolchains.any? { |toolchain| toolchain == "gcc" }
-          c.flags += %w(-g3 -O0)
+          c.flags += %w(-g3 -O2)
         end
       end
       @mrbc.compile_options += ' -g'
